lib/src/react_client/synthetic_events.dart

```diff
@@ -0,0 +1,325 @@
+import 'dart:html';
+
+import 'package:react/react.dart';
+import 'package:react/src/react_client/synthetic_event_wrappers.dart' as events;
+
+/// A mapping from converted/wrapped JS handler functions (the result of [_convertEventHandlers])
+/// to the original Dart functions (the input of [_convertEventHandlers]).
+final Expando<Function> originalEventHandlers = new Expando();
```

Comment:
This is accessed via both the `utils.dart` files (lib/react_client/utils & lib/src/react_client/utils). Seemed most fitting to group it with the events, though there may be somewhere better